content/drone-plugins/drone-gcf/index.md

```diff
@@ -0,0 +1,82 @@
+---
+date: 2016-01-01T00:00:00+00:00
+title: Google Cloud Functions
+author: oliver006
+tags: [ publish, google, gcp, gcf ]
+logo: google_gcf.svg
+repo: oliver06/drone-gcf
+image: plugins/gcf
```

Comment:
It's not published in the plugins hub organization, so this should be oliver006/drone-gcf

Reply:
done